models/tasks.js

```diff
@@ -341,6 +341,26 @@ const overdueTasks = async (overDueTasks) => {
     throw err;
   }
 };
+
+/**
+ * Fetch all overdue tasks objects
+ * @param overdueTasks <Array>: tasks which are overdue
+ * @return {Promsie<Array>}
+ */
+const getAllOverDueTasks = async () => {
```

Comment:
how about just calling it getOverDueTasks (since its already plural i dont think we need the all keyword)

Reply:
ok